spikeinterface/extractors/mclustextractors.py

```diff
@@ -0,0 +1,94 @@
+from pathlib import Path
+import re
+import numpy as np
+
+from spikeinterface.core import (BaseSorting, BaseSortingSegment)
+from spikeinterface.core.core_tools import define_function_from_class
+
+
+class MClustSortingExtractor(BaseSorting):
+    """Load MClust sorting solution as a sorting extractor.
+
+    Parameters
+    ----------
+    folder_path : str or Path
+        Path to folder with t files.
+    sampling_frequency : sampling frequency
+        sampling frequency in Hz.
+    raw_to_sec: float or None
```

Comment:
is this a `sampling_period`? Can we find a better namie @ferchaure? :)

Reply:
It could be a sampling period, but not necessarily , the sorter could be run using files where the "raw time" associated with each snippet is in an unknown unit. That "raw time" could be tends of ms, samples, micro seconds, almost anything really. I asked the author [here](https://github.com/NeuralEnsemble/python-neo/pull/1150#issuecomment-1212594023) (That change is not required, it's just to improve the header text)

For that  cases, I need an additional input to know the unit of the "raw time"... It could be raw time to samples as well if you prefer. About the name "raw_time_to_seconds"/"raw_time_to_samples"?

On the other hand, the sampling_frequency  could have another value, and it's the only required input for the usual files and I need it to implement a get_sampling_frequency().

Reply:
The raw time is assumed to be constant, right? so it can be considered a sampling periot for the traces (if I understand correctly). For consistency, what about using `sampling_frequency_raw=None`:
- if None, then we assume that raw_to_sec is `1 / sampling_frequency`, otherwise it's `1/ smapling_frequency_raw`

Reply:
done